### PR TITLE
doc/cephadm: fixing cluster purging section

### PR DESCRIPTION
**Pull request by rkachach**
https://tracker.ceph.com/issues/54018

Signed-off-by: Redouane Kachach <rkachach@redhat.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
could you add a sentence or two to your commit message why you're doing this?

File: doc/cephadm/operations.rst [hunk 1]
```diff
@@ -524,18 +524,18 @@ Purging a cluster
 
 .. danger:: THIS OPERATION WILL DESTROY ALL DATA STORED IN THIS CLUSTER
 
-In order to destroy a cluster and delete all data stored in this cluster, pause 
-cephadm to avoid deploying new daemons.
+In order to destroy a cluster and delete all data stored in this cluster, disable
+cephadm to stop all orchestration operations (so we avoid deploying new daemons).
 
-.. prompt:: bash #
+.. prompt:: bash (cephadm shell session)#
```
Comment (adk3798) [inline doc/cephadm/operations.rst:530]: in the readthedocs render this (and the other prompt) aren't showing up

image

possibly caused by putting "(cephadm shell session)" here

**Review by adk3798 — CHANGES_REQUESTED** [review]

File: doc/cephadm/operations.rst (hunk 1 above)
Comment (sebastian-philipp) [reply doc/cephadm/operations.rst:530]: https://ceph--44810.org.readthedocs.build/en/44810/cephadm/operations/#purging-a-cluster

Comment (adk3798) [reply doc/cephadm/operations.rst]: Fixed now 
image

**Review by adk3798 — APPROVED** [review]

**Pull request merged**